apps/server/src/controllers/v1/users.ts

```diff
@@ -24,6 +24,12 @@ router.post('/login', async (req: Request, res: Response) => {
     })
 })
 
+router.post('/logout', async (req: Request, res: Response) => {
+    const { accessToken, refreshToken } = req.body
+    await expireJwt({ accessToken, refreshToken })
```

Comment:
authorization 헤더의 jwt 를 넣어주면 됩니다
```suggestion
    await expireJwt(jwt)
```